code/modules/vending/wardrobes.dm

```diff
@@ -228,6 +257,10 @@
 					/obj/item/clothing/under/rank/chef = 1,
 					/obj/item/clothing/head/chefhat = 1,
 					/obj/item/reagent_containers/glass/rag = 1)
+	contraband = list(/obj/item/storage/box/condimentbottles = 3,
```

Comment:
doesn't belong in a wardrobe but would be a great addition to the kitchenmate or whatever it's called